docs/js/modal.md

```diff
@@ -88,38 +87,75 @@ new Modal(document.querySelectorAll('[data-modal]'), options);
 
 By default, modal don't provide any trigger to open it. Where's an example of how to create a button to open modal.
 
-```html
-<button data-trigger>Open Modal</button>
-```
+<div class="example example-code">
+  <button class="button button-primary" data-trigger>Open Modal</button>
+</div>
 
 ```js
-let modal = $('[data-modal]').modal(),
-    trigger = $('[data-trigger]');
+let modal = $('[data-modal-trigger]').modal().data('modal');
+let trigger = $('[data-trigger]');
 
 trigger.on('click', () => {
   modal.show();
 });
 ```
+<div data-modal-trigger class="hide">
+  <div class="row">
+    <div class="col-xs-12">
+      <h2>Trigger Modal example</h2>
+      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+    </div>
+  </div>
+</div>
+
 Once you called `show()`, `modal()` bind the close icon and `esc` keyboard key to hide modal.
 
-If you want that a button inside `.modal` close itself, add option `triggerClose` with a string selector. Modal will delegate `click` to selector and call `hide()` when click is fired.
+To have a default open and close button, you can use `triggerOpen` and `triggerClose` and modal will delegate `click` event to these selector and call `show()` or `hide()` when click is fired. Example:
 
 ```js
-let modal = $('[data-modal]').modal({
-      triggerClose: '.any-selector',
-      triggerOpen: '[data-trigger="open"]'
-    })
+$('[data-modal]').modal({
+  triggerClose: '.any-selector',
+  triggerOpen: '[data-trigger="open"]'
+});
 ```
-Working Example:
 
-<button class="button button-primary" data-trigger="open">Open Modal</button>
+## Set Modal size
+
+The modal has three predefined sizes that can be chosen, small, medium or large.
```

Comment:
Remove `...that can be chosen` e deixa só as opções precedidas de `:`
Coloca vírgula antes de `or`